ublox_gps/config/rover.yaml

```diff
@@ -37,17 +37,44 @@ ublox_gps_node:
       min_dur: 300              # Survey-In Minimum Duration [s]
       acc_lim: 3.0              # Survey-In Accuracy Limit [m]
     
-    nmea:
-      high_prec: true
-
+​
+      nmea:
+        set: true
+        version: 65
+        num_sv: 8
+        sv_numbering: 1
+        compat: true
+        consider: true
+        limit82: false
+        high_prec: true
+        filter:
+          pos: false
+          msk_pos: false
+          time: false
+          date: false
+          gps_only: false
+          track: true
+        gnssToFilter:
+          gps: false
+          sbas: false
+          qzss: false
+          glonass: false
+          beidou: false
```

Comment:
Does all this need to be declared as false explicitly? My guess is that False is their default value